### PR TITLE
Feature/interactions to raw

### PR DESCRIPTION
**Pull request by feldlime**
- Added methods for conversion `Interactions` to raw form and for getting raw interactions from `Dataset`

### CODE REVIEW

File: rectools/dataset/interactions.py [hunk 1]
```diff
@@ -125,8 +125,6 @@ def get_user_item_matrix(self, include_weights: bool = True) -> sparse.csr_matri
         """
         Form an user-item CSR matrix based on `interactions` attribute.
```
Comment (blondered) [inline rectools/dataset/interactions.py:126]:
```suggestion
        Form a user-item CSR matrix based on interactions data.
```

File: rectools/dataset/interactions.py
Comment (blondered) [inline rectools/dataset/interactions.py:1]: Let's also fix docstring for `Interactions` class. https://github.com/MobileTeleSystems/RecTools/blob/a7e6407a7a969f72d22f2806eea74687b9b82ece/rectools/dataset/interactions.py#L30  `Structure to store info about user-item interactions.`

File: rectools/dataset/interactions.py [hunk 2]
```diff
@@ -152,3 +150,42 @@ def get_user_item_matrix(self, include_weights: bool = True) -> sparse.csr_matri
...
+        Convert itself to `pd.DataFrame` with replacing internal user and item ids to external ones.
+
+        Parameters
+        ----------
+        user_id_map : IdMap
+            User id map that has to be used for converting internal user ids to external ones.
+        item_id_map : IdMap
+            Item id map that has to be used for converting internal item ids to external ones.
+        include_weight : bool, default ``True``
+            Whether include weight column into resulting dataset or not.
```
Comment (blondered) [inline rectools/dataset/interactions.py:171]: Needed both here, for next attribute and for dataset class.
```suggestion
            Whether to include weight column into resulting table or not.
```

File: rectools/dataset/interactions.py
Comment (blondered) [inline rectools/dataset/interactions.py:1]: Can we please fix the https://github.com/MobileTeleSystems/RecTools/blob/a7e6407a7a969f72d22f2806eea74687b9b82ece/rectools/dataset/interactions.py#L73

**Review by blondered — CHANGES_REQUESTED** [review]

**Comment by blondered** [comment]
Let's rename `to_raw` -> `to_external`

File: rectools/dataset/interactions.py (hunk 1 above)
Comment (feldlime) [reply rectools/dataset/interactions.py]: Fixed

File: rectools/dataset/interactions.py
Comment (feldlime) [reply rectools/dataset/interactions.py:1]: Fixed

File: rectools/dataset/interactions.py (hunk 2 above)
Comment (feldlime) [reply rectools/dataset/interactions.py]: Fixed

File: rectools/dataset/interactions.py
Comment (feldlime) [reply rectools/dataset/interactions.py:1]: Let's do it in a separate PR

**Review by blondered — APPROVED** [review]

**Pull request merged**